web/spec/supabase.yml

```
@@ -1451,7 +1451,7 @@ pages:
             .subscribe()
           ```
       - name: Listening to row level changes
-        description: You can listen to individual rows using the format `{table}:{col}=eq.{val}` - where `{col}` is the column name, and `{val}` is the value which you want to match.
+        description: You can listen to individual rows using the format `{table}:{col}=eq.{val}` - where `{col}` is the column name, and `{val}` is the value which you want to match. Only filtering by a single column and only using `eq`, `neq`, `lt`, `lte`, `gt`, `gte` operators is available now. Keep in mind, Supabase would not send an update event if `{col}` changed from `{val}` to `{val2}`.
```

```suggestion
        description: You can listen to individual rows using the format `{table}:{col}=eq.{val}` - where `{col}` is the column name, and `{val}` is the value which you want to match. Only filtering by a single column and only using `eq`, `neq`, `lt`, `lte`, `gt`, `gte` operators are available. Keep in mind, Supabase would not send an update event if `{col}` changed from `{val}` to `{val2}`.
```

Thanks for this @tchief - a munch needed update.

Can you clarify this sentence? 

```
Keep in mind, Supabase would not send an update event if `{col}` changed from `{val}` to `{val2}`.
```

This only pertains to the `eq` filter right?

---

@kiwicopple This sentence is for `eq` filter, correct.

I expected it do be true for other filters, so just double-checked that. And it seems that `${table}:${column}=gte.${value}` doesn't produce realtime updates for filters others than `eq` at all.

I use it this way
```
supabase.from<T>(`${table}:${column}=gte.${value}`)
  .on('DELETE', handleDelete)
  .on('UPDATE', handleUpdate)
  .subscribe();
```

Maybe syntax is different for other operators. However, with `neq` I expect it to be the same, and it doesn't work.
Let me know, please, if there's another syntax to try other filters.

Otherwise I'll update PR and remove all the other operators list except `eq`.

---

Perhaps we can update it to 

```yaml
description: | 
    You can listen to individual rows using the format `{table}:{col}=eq.{val}` - where `{col}` is the column name, and `{val}` is the value which you want to match. 
    
    Only filtering by a single column and only using `eq`, `neq`, `lt`, `lte`, `gt`, `gte` operators are available. 
    When using the `eq` and `neq` filters, keep in mind that an event will not be sent if a column is updated to a value that no longer matches your filter.

```

---

That's great!

One thing to mention, unfortunately Supabase doesn't indeed yet supports all the other operators for Realtime.
Here's the [link](https://github.com/supabase/realtime/issues/132#issuecomment-1089121668).

---

(so actually all of those filters should work)

---

Should I enable WALRUS in my Supabase dashboard?
Using latest dependency `"@supabase/supabase-js": "^1.33.3"` still cannot make it send updates/deletes with 'gte'.

---

Added code with steps to reproduce [here](https://github.com/tchief/supabase-js/commit/7abf9983cf19024e0937c8dc5baabda6379c5887), based on example from `supabase-js` repo.
Here's testing process

https://user-images.githubusercontent.com/729374/162426231-7a6071cd-aa82-4b51-8920-647739964ecc.mp4